### ISSUE TITLE
Minimap is bugged with agility shortcuts

### ISSUE THREAD

**Issue by daviscook477** [runelite/runelite#6674]
**Describe the bug**
On the world map agility shortcuts are properly indicated as either able to be used or unable to be used by the player's agility level by either having a red "no" circle around them or not having the circle. However, on the minimap all agility shortcuts, no matter the level, will display as being able to be used.

**To Reproduce**
1. Go to an agility shortcut for which you do not have a high enough level (for me it was the blue dragon shortcut in taverly dungeon)
2. open the world map and note that agility shortcut icon
3. look at your minimap anad note the agility shortcut icon

**Expected behavior**
The minimap agility icon should indicate if it is not able to be used.

**Screenshots**
![agi_seems_valid](https://user-images.githubusercontent.com/10566967/48975367-99923500-f03c-11e8-9f1d-7663107bc2d1.png)
![agi_not_actually_valid](https://user-images.githubusercontent.com/10566967/48975368-9b5bf880-f03c-11e8-8236-afbab8e7c525.png)

**Environment (please complete the following information):**
 - OS: Windows
 - RuneLite version: [e.g 1.4.6]
 - Launcher version: [e.g 1.5.2]

**Comment by Nightfirecat** [comment]
That's because we only overlay over the world map, not the minimap. You'll notice that all the custom icons we add to the world map (teleport icons, etc.) are also not displayed on the minimap.

**Comment by daviscook477** [comment]
Fair enough. Since I know nothing about this part of the code, is it feasible to overlay the minimap in addition to overlaying the world map?
